go.work

```diff
@@ -0,0 +1,6 @@
+go 1.22.0
```

Comment:
should this match with the root `go.mod` file version?

Reply:
updated 👍